README.md

```diff
@@ -46,6 +46,7 @@ Debian system.
 ##### Install (upgrade) required Python libraries
 
 	$ sudo pip install --upgrade trappy bart-py devlib
+	$ sudo pip install netifaces
```

Comment:
well, we should define a reasonable strategy... so far I've used apt-get for python modules which are not direct LISA submodules. That's why there is a pip install command for just trappy&friends...